src/orquestra/sdk/_ray/_dag.py

```diff
@@ -258,6 +260,10 @@ def _(imp: ir.PythonImports):
 
 @_pip_string.register
 def _(imp: ir.GitImport):
+    # Only download Git imports if a specific environment variable is set
+    # Short circuit the Git import otherwise
+    if os.getenv(RAY_DOWNLOAD_GIT_IMPORTS_ENV) is None:
+        return []
```

Comment:
Sometimes it's easier to set an env variable to `"0"` or `""` than to unset it fully.

```suggestion
    # Only download Git imports if a specific environment variable is set
    # Short circuit the Git import otherwise
    if os.getenv(RAY_DOWNLOAD_GIT_IMPORTS_ENV) != "1":
        return []
```